### ISSUE TITLE
Fishing comp STV

### ISSUE THREAD

**Issue by tobyewald** [The-Barrens-org/Bugtracker#42]
Fishing comp doesnt run on its  schedule of 2-4pm server time, could we please get it changed to this time to avoid confusion and so a few more of us are able to attend.

**Comment by Nolin7777** [comment]
Do you know how many hours this event is off?

**Comment by tobyewald** [comment]
Yes. The event is off by 9 hours. It should start be from 2-4pm server time instead of 11pm-1am that is it at the moment. Cheers for your help

Sent from my iPhone

On 12 Jan 2019, at 7:04 am, Nølin <notifications@github.com<mailto:notifications@github.com>> wrote:


Do you know how many hours this event is off?
